### PR TITLE
Update React.js README.md

### PR DESCRIPTION
**Pull request by john-raymon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
-    - Contains a plain HTML/CSS/SASS/Vanilla JS implemenation.
-  - `React.js`
-    - Contains an abstracted version of the Master branch using React.js, Redux, CSS/SASS, JSX
+This demo-store uses [Commerce.js](https://github.com/chec/commerce.js), an eCommerce client-side 
+JavaScript SDK. [View the live demo](https://chec.github.io/example.checkout.v2/#/).
+
+# Spin up a local demo
+Follow the instructions below to [install the Chec CLI](https://github.com/chec/cli#usage).
+
+## Use [Chec CLI](https://github.com/chec/cli) to create copy of source code on your local computer
```
Comment (robbieaverill) [inline README.md]:
```suggestion
## Install source code using the [Chec CLI](https://github.com/chec/cli)
```

To keep the headings a bit shorter

File: README.md [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
+Use the Chec CLI to create copy of source code on your local computer. First follow the instructions to install the Chec CLI.
```
Comment (robbieaverill) [inline README.md]:
```suggestion
Use the Chec CLI to create copy of the source code on your computer. First follow the instructions to install the Chec CLI.
```

File: README.md [hunk 3]
```diff
@@ -1,10 +1,28 @@
...
+
+Once installed login using the Chec login command. Then run the Chec demo-store command and select the `react-shoe-store`.
+
+If authenticated successfully (tip: use Chec whoami) your Chec dashboard should now include two sample products, seeded by the Chec CLI, and a demo store directory in your root.
```
Comment (robbieaverill) [inline README.md]:
```suggestion
If authenticated successfully (tip: use `chec whoami`) your Chec Dashboard should now include two sample products, seeded by the Chec CLI, and your terminal should have a `react-shoe-store` directory.
```

File: README.md [hunk 4]
```diff
@@ -1,10 +1,28 @@
...
+
+## Clone & Use NPM script `seed`
```
Comment (robbieaverill) [inline README.md]:
```suggestion
## Clone & use NPM script `seed`
```

File: README.md [hunk 5]
```diff
@@ -1,10 +1,28 @@
...
+You can also skip the use of the `Chec CLI` and instead clone the repo, switch to the `react.js` branch, make a copy of `.env.example` into `.env`—setting the variables needed.
```
Comment (robbieaverill) [inline README.md]:
```suggestion
You can also skip the use of the Chec CLI and instead clone the repo, switch to the `react.js` branch, make a copy of `.env.example` into `.env` and setting the variables needed. 
```

**Review by robbieaverill — COMMENTED** [review]
Nice, some minor suggestions

**Comment by john-raymon** [comment]
@robbieaverill Thanks Robbie! Squashed the commits, good to merge?

**Review by robbieaverill — APPROVED** [review]

**Pull request merged**